vro-types/vrotsc-annotations/index.d.ts

```diff
@@ -61,14 +61,59 @@ interface VroPolicyTemplate {
 	path?: string;
 	type?: VroPolicyTemplateType;
 	version?: string;
+	templateVersion?: "v1" | "v2";
+	variables?: {
+		[name: string]: string | PolicyAttribute;
+	};
+	elements?: {
+		[name: string]: PolicyElement
+	};
 	schedule?: {
 		periode: VroPolicyTemplateScehdulePeriod;
 		when: string;
 		timezone: string;
 	};
 }
 
+type PolicyElement = {
+	type: VroPolicyTemplateTypeV2;
+	events?: {
```

Comment:
Custom events may be defined by exported plugins. I believe the correct typere here would be:

```ts
type PolicyElement = {
	type: VroPolicyTemplateTypeV2;
	events?: {
		[event: string]: string | PolicyWorkflowInfo;
	};
	schedule?: {
		periode: VroPolicyTemplateScehdulePeriod;
		when: string;
		timezone: string;
	};
};
```

Reply:
Changed type as suggested.

Reply:
@akashayyappan It would be easier if you create the merge request via branch created directly from this main branch without need to fork to a private repository. This would make the merging and maintaining the code more streamlined.